/usr/bin/python: can't open file '/home/nvdia/kernel_panic/core/spyder/scripts/logger.py': [Errno 2] No such file or directory
